### ISSUE TITLE
Image is not rendered when using borderRadius & resizeMode [Android][RN 0.46.2]

### ISSUE THREAD

**Issue by Mindaugas-Jacionis** [facebook/react-native#15053]
### Environment
- react-native: 0.46.2
- node: 6.9.1
- npm: 3.10.8
- yarn: 0.17.9
- Target Platform: Android
- Development Operating System: macOS Sierra v10.12.13

### Steps to Reproduce
1. Create `Image`
2. Use `resizeMode` and `borderRadius` for styling of the image.

### Expected Behavior
Image should be rounded and follow resize prop instructions('cover', 'contain', 'stretch', 'repeat', 'center').

### Actual Behavior
Instead image simply isn't rendered. If you remove `borderRadius` or `resizeMode`. All works as intended.

### Reproducible Demo
1. In terminal run `react-native init testImg`
2. open project in text editor and change `index.android.js` to the one below:
```
import React, { Component } from 'react';
import { AppRegistry, StyleSheet, View, Image } from 'react-native';

export default class testImg extends Component {
  render() {
    return (
      <View style={styles.container}>
        <Image
          style={styles.avatar}
          source={{uri: 'https://pbs.twimg.com/profile_images/831993825635745796/HnVmB0-k.jpg'}}
        />
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: '#F5FCFF',
  },

  avatar: {
    width: 70,
    height: 70,
    borderRadius: 35,
    // resizeMode: Image.resizeMode.contain // uncomment to see the issue
  }
});

AppRegistry.registerComponent('testImg', () => testImg);
```
3. In project folder run `react-native run-android`, you will see working image, but if you uncomment `resizeMode` in the code - image won't render.

**Comment by ItsNoHax** [comment]
Holy mother of god, I spent 5 hours debugging at work why my image wouldn't load anymore...
I'm fuming that this is the cause.

**Comment by nehvaleem** [comment]
also having exact same issue

**Comment by ismjt** [comment]
thanks god, you save my time! Is there any solution?

**Comment by datnguyen92** [comment]
I have the same issue on RN 0.46.4 as well, do we have plan to fix this?

**Comment by ekkucukarpaci** [comment]
I'm also having this issue, tested only on Android. `borderRadius` only seems to work when `resizeMode` is `'cover'`. Maybe e0bdb7cbcfa6ff20daae8cdd9cd09b95802c74bb could be related?

**Comment by wielrls** [comment]
Ditto. My workaround in the short term is to use `borderRadius` with `resizeMode={'cover'}`.

**Comment by rdixonbhw** [comment]
It appears to have been caused by this commit https://github.com/facebook/react-native/commit/e0bdb7cbcfa6ff20daae8cdd9cd09b95802c74bb#diff-983c85ec5691d0cd750127399edff526

**Comment by rendomnet** [comment]
If you will use resizeMode: stretch then border radius will be **deformed**

**Comment by shoppy999** [comment]
You don't reply to the issues, but wait them out and close due to inactivity. Brilliant strategy.

**Comment by rdixonbhw** [comment]
@shoppy999 This was fixed months ago in 0.47

**Comment by raynor85** [comment]
It seems like Images don't work well with the new border properties at all. borderTopWidth, borderTopColor and all the variants don't work at all.

**Comment by aligfl** [comment]
so was this fixed in RN >= 0.47?
